gradle/libs.versions.toml

```diff
@@ -45,6 +45,8 @@ layoutlib-native-macArm = { module = "app.cash.paparazzi:layoutlib-native-macarm
 layoutlib-native-macOsX = { module = "app.cash.paparazzi:layoutlib-native-macosx", version.ref = "layoutlib" }
 layoutlib-native-windows = { module = "app.cash.paparazzi:layoutlib-native-win", version.ref = "layoutlib" }
 
+material = { group = "com.google.android.material", name = "material", version = "1.8.0" }
```

Comment:
while #380 is definitely a recent example of this bug, I think the issue ultimately is that Build.FOO references weren't working, so I'd like to suggest:

1) reverting these test changes
2) adding a plugin test that is something like this:

```
class BuildTest {
  @get:Rule
  val paparazzi = Paparazzi()

  @Test
  fun build() {
    assertThat(Build.SOME_VALUE).isEqualTo("whatevs")
    // ... pick a few values per known issues
  }
}
```